doc/20-REST-API.md

```diff
@@ -0,0 +1,31 @@
+# REST API
+
+The Icinga Notifications module provides a REST API that allows you to manage notification-related resources programmatically.
+
+With this API, you can:
+- Manage **contacts** and **contact groups**
+- Read available **notification channels**
+
+This API enables easy integration with external tools, automation workflows, and configuration management systems.
+
+---
```

Comment:
These look weird, how did you come up using them? I'd rather remove them

<img width="261" height="140" alt="image" src="https://github.com/user-attachments/assets/7f249bda-2d64-4654-bf5d-a918fa48165f" />